src/net.cpp

```diff
@@ -399,6 +401,38 @@ static CAddress GetBindAddress(SOCKET sock)
     return addr_bind;
 }
 
+bool derive_bip324_keys(ECDHSecret&& ecdh_secret, const Span<uint8_t> initiator_hdata, const Span<uint8_t> responder_hdata, BIP324Keys& derived_keys)
```

Comment:
Why snake case for function name `derive_bip324_keys` here?

Reply:
Oh oops. I don't know why I did that. Updated.